07/reshape-a-matrix.checkpoint.md

```diff
@@ -401,7 +208,58 @@ def reshape_matrix(matrix, r, c):
 
 <!-- ======================= END CHALLENGE ======================= -->
 
-## Reminders
+### !challenge
+
+* type: short-answer
+* id: 976fd220-e3f3-41f4-adb2-908e30d1e363
+* title: What is the time complexity of your solution?
+* points: 0
+* topics: Big-O, Python, lists, matrices
+
+##### !question
+
+What is the time complexity of your solution?
+
+##### !end-question
+
+##### !placeholder
+
+Time Complexity?
+
+##### !end-placeholder
+
+##### !answer
+
+/.+/
+
+##### !end-answer
 
-- Finish this assignment individually.
-- If you'd like a full list of our tips, hints, and notes, please bring up the document "[About Problem Solving Exercises](../about-pses/about-pses.md )" in unit.
+### !end-challenge
+
+### !challenge
+
+* type: short-answer
+* id: 7bb7f05e-257c-43ba-a811-8900cfd622e9
+* title: What is the space complexity of your solution?
+* points: 0
```

Comment:
Talked to some instructors, and we actually decided 1 point it better. sorry for the back and forth!

```suggestion
* points: 1
```